Gemfile

```diff
@@ -2,4 +2,4 @@ source 'https://rubygems.org'
 
 gem 'jekyll'
 gem 'jekyll-redirect-from'
-gem 'rouge'
+gem 'rouge', :git => 'https://github.com/kylef/rouge', :branch => 'kylef/apiblueprint-lexer'
```

Comment:
Sorry to jump in, but I am definitely against having references to personal repositories. Could you please fork it under apiaryio account?